.github/actions/build-test-environment/action.yml

```
@@ -31,7 +31,7 @@ runs:
           echo "Running spikeinterface dev version"
           pip uninstall -y neo
           pip uninstall -y probeinterface
-          pip install git+https://github.com/NeuralEnsemble/python-neo
+          pip install git+https://github.com/h-mayorquin/python-neo.git@improve_mearec
```

pip install git+https://github.com/NeuralEnsemble/python-neo
```suggestion
          pip install git+https://github.com/NeuralEnsemble/python-neo
```